pkg/postgres/query/query.go

```diff
@@ -71,3 +71,98 @@ func checkFieldValueType(field filter.RequestField) (valueIsList bool, valueList
 	}
 	return valueIsList, valueList, nil
 }
+
+func likeOperatorCondition(
+	field filter.RequestField, valueIsList bool, negate bool, beginsWith bool,
```

Comment:
Maybe add a comment that `beginsWith` false means `contains` as this is not immeditealy obvious

Reply:
Added a comment to the `likeOperatorCondition`

Reply:
Added some comments:
`// if beginsWith is false, the condition should check for "contains" and use containsAsLikePattern()`